CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 
-## Fleet 4.17.0 (July 8, 2022)
```

Comment:
@Desmi-Dizney heads up looks like you accidentally deleted this header from the changelog.

Reply:
@zwass, I created https://github.com/fleetdm/fleet/pull/6902 to resolve this.